hacks/ops-pod

```diff
@@ -111,7 +111,7 @@ if [[ -n ${node_of_pod} ]]; then
 else
   echo "Node name provided ..."
   # shellcheck disable=SC2076
-  if [[ $(kubectl get nodes -o jsonpath='{.items[*].metadata.name}') =~ " ${node} " ]]; then
+  if kubectl get nodes -o jsonpath='{.items[*].metadata.name}' | grep -q "\b${node}\b"; then
     echo -e "Deploying ops pod on ${node}\n"
   else
     echo -e "Error: node ${node} does not exist in the cluster.\n"
```

Comment:
Can we use this PR to also add an `exit 1` after the `print_usage` call so that we do not try to start up a pod that will never become `Ready`?